lambdas/src/contact-retrieve-transcript/exportTranscript.ts

```diff
@@ -0,0 +1,51 @@
+import { Twilio } from 'twilio';
+
+export type ExportTranscriptParams = {
+  accountSid: string;
+  authToken: string;
+  serviceSid: string;
+  channelSid: string;
+};
+
+export type ExportedMessage = {
+  sid: string;
+  dateCreated: Date;
+  from: string;
+  body: string;
+  index: number;
+  type: string;
+  media: any;
+};
+
+export type ExportTranscriptReturn = ExportedMessage[];
+
+export const exportTranscript = async ({
+  accountSid,
+  authToken,
+  channelSid,
+  serviceSid,
+}: ExportTranscriptParams): Promise<ExportTranscriptReturn> => {
```

Comment:
Since `twilio` is a typed library, you don't really need to be explicit about this return type, you could let TS infer it from you. If you remove `: Promise<ExportTranscriptReturn>` (and the type unused types), at least VSCode infer that
```
const exportTranscript: ({ accountSid, authToken, channelSid, serviceSid, }: ExportTranscriptParams) => Promise<{
    sid: string;
    dateCreated: Date;
    from: string;
    body: string;
    index: number;
    type: string;
    media: any;
}[]>
```
That also solves the comment below to address the returned structure type.